app/models/address.rb

```diff
@@ -2,6 +2,7 @@
 class Address
   include Mongoid::Document
   include Mongoid::Timestamps
+  include Mongoid::Attributes::Dynamic
```

Comment:
Let's take this out, it would let in any attribute at all.

Reply:
Sure